library/neptune-triton/futhark-version.txt

```diff
@@ -1,4 +1,5 @@
-Futhark 0.16.2
+Futhark 0.16.0
+git: HEAD @ 965e8eb (Fri Jul 3 14:53:26 2020 +0200)
```

Comment:
For your future reference, @dignifiedquire, this happens because you built from source instead of using the release version. For any given release number, the code should be otherwise equivalent.

Reply:
Because the last digit in the version number 0, this was compiled with a prerelease version of what eventually became Futhark 0.16.1.  Considering how many bugfixes were motivated by neptune-triton, I strongly recommend not using anything earlier than 0.16.1.

Reply:
Ah, good catch. I wasn't paying attention to the version number when I wrote that comment.

In any case, this PR was superseded by #20 – which does not regress the Futhark version and instead kept 0.16.2 in order not to introduce any compiler-related changes.

I'm going to close this PR for the sake of clarity.